packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -134,11 +136,14 @@ export default class LocusInfo {
    * @returns {Array} updatedParticipants
    */
   updateParticipants(participants, deltaParticpantFlag) {
-    this.emitChanges(deltaParticpantFlag ? 'LOCUS_INFO_DELTA_PARTICIPANTS' : 'LOCUS_INFO_FULL_PARTICIPANTS', {
-      participants,
-      self: this.self,
-      host: this.host
-    });
+    this.emitChanges(
+      deltaParticpantFlag ? EVENTS.LOCUS_INFO_DELTA_PARTICIPANTS : EVENTS.LOCUS_INFO_FULL_PARTICIPANTS,
```

Comment:
do we need different event names/types for delta vs full? Feels like maybe we could capture this with just one event.

Reply:
yeah, i can change it now that I have them routing to the same code, previously they were split